src/style.css

```diff
@@ -1 +1,92 @@
-/* add styles here */
+body {
+  /* stylelint-disable-next-line font-family-name-quotes */
+  font-family: 'Avenir', Arial, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. It's a good practice to specify a generic family as a fallback, such as `sans-serif` for Avenir in this case. For example: `font-family: 'Avenir', Arial, sans-serif;`